projects/plugins/boost/app/lib/minify/functions-service.php

```diff
@@ -238,7 +238,7 @@ function jetpack_boost_page_optimize_build_output() {
 					function ( $match ) {
 						global $pre_output;
 
-						if ( 0 === strpos( $pre_output, '@charset' ) ) {
+						if ( 0 === strpos( (string) $pre_output, '@charset' ) ) {
```

Comment:
I think a guard clause for non-strings would be better here

Reply:
I've reworked how the variable is used in aff554fb6b5d1a2c21116df82d3c8456db55e98a, so casting is no longer necessary.